sdk/storage/azure_storage_blob/README.md

```diff
@@ -1,3 +1,144 @@
-# sdk/storage/azure_storage_blob
+# Azure Storage Blob client library for Rust
 
-TODO TODO TODO
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob storage is optimized for storing massive amounts of unstructured data, such as text or binary data.
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [REST API documentation] | [Product documentation]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Storage Blob client library for Rust with [cargo]:
+
+```sh
```

Comment:
You use both `sh` and `bash` below. You should be consistent. The rest of us are using `sh`, so lets stick with that.

The only exception is when you have a line with just `az` then use `azcli`. IIRC, it gets special treatment for learn.microsoft.com if/when they ingest these.